sympy/core/symbol.py

```diff
@@ -205,10 +205,22 @@ def numbered_string_incr(s, start=0):
 
 class Symbol(AtomicExpr, Boolean):
     """
+    A Symbol is an expression of a mathematical variable using a
```

Comment:
A docstring should begin with a single line single sentence statement of what the object/function is/does.